src/App/analytics.js

```diff
@@ -1,6 +1,7 @@
 import Analytics from 'electron-ga';
 const analytics =  new Analytics('UA-22170471-17');
-const isDev = require('electron-is-dev');
+const isDev = true;
```

Comment:
perhaps a comment with the same message in the code would be helpful, so at least this is not forgotten.

suggested:
// disable analytics until we comply with GA policy